prody/proteins/interactions.py

```diff
@@ -180,21 +229,69 @@ def get_energy(pair, source):
     
     lookup = pair[0]+pair[1]
     
-    return data[np.where(np.array(aa_pairs)==lookup)[0]][0][2:][np.where(np.array(sources)==source)][0]
+    try:
+        data_results = data[np.where(np.array(aa_pairs)==lookup)[0]][0][2:][np.where(np.array(sources)==source)][0]
+    except TypeError:
+        raise TypeError('Please replace non-standard names of residues with standard names.')
+
+    return data_results
+
+
+def checkNonstandardResidues(atoms):
```

Comment:
Probably this should return True or False instead of LOGGER outputs. Also, it doesn't seem to be used anywhere

Reply:
actually, I don't think this function works at all. I'm reading through it again